recipes/sctk/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sctk" %}
+{% set version = "2.4.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 43bd2bfe60e0ad04a563ddef025e84fca4c073f8f7c02f056b59fb8e2ec4f022
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - m2-base  # [win]
+
+test:
+  source_files:
+    - makefile
+    - src/**
+  commands:
+    - make check -C $SRC_DIR  # [not win]
```

Comment:
If you want to run make check, please do it as part of the build script instead of including a copy of the entire source tree in the package.

My understanding is that this package is a collection of CLI tools, so the tests section should be calling "$ tool --help" or like "$ tool --version", to check that they return without error.

Reply:
I understand. Unfortunately the developer did not add either option. What would be the recommendation in this case, simply checking for the existence of the files?